.github/workflows/build-mudlet.yml

```diff
@@ -39,12 +39,19 @@ jobs:
             gcc_compiler_version: 9
             qt: '5.14.2'
           - os: macos-12
-            buildname: 'macos / c++, lua tests'
+            buildname: 'macos (x86_64) / c++, lua tests'
             triplet: x64-osx
             compiler: clang_64
             qt: '5.14.2'
             deploy: 'deploy'
             run_tests: 'true'
+          - os: macos-14
+            buildname: 'macos (arm64) / c++, lua tests'
+            triplet: arm64-osx
+            compiler: clang_64
+            qt: '5.15.13'
```

Comment:
:thinking: If that is the version that is currently in use does that mean we need to up the Qt version in the other MacOS builds - ***in a different PR***?

Reply:
This is where we'd do it, yep